smartsim/_core/control/manifest.py

```diff
@@ -94,9 +90,11 @@ def all_entity_lists(self) -> t.List[EntityList]:
         _all_entity_lists: t.List[EntityList] = []
         _all_entity_lists.extend(self.ensembles)
 
-        db = self.db
-        if db is not None:
-            _all_entity_lists.append(db)
+        dbs = self.dbs
+
+        if dbs is not None:
```

Comment:
As now `dbs` is of type `t.List[Orchestrator]` it cannot be `None`. I think you can save some lines and enhance core readability by simply iterating over `self.dbs`, like 
```python
 for db in self.dbs:
```